### PR TITLE
Small refactoring

### PR DESCRIPTION
**Pull request by hparfr**
Reuse function get_repo_key_from_spec

### CODE REVIEW

File: ak/ak_build.py [hunk 1]
```diff
@@ -225,6 +225,8 @@ def main(self, *args):
         config_file = self.config
         self._ensure_viable_installation(config_file)
         self._generate_links(config_file)
+        force_directory = get_repo_key_from_spec(self.directory)
```
Comment (bealdav) [inline ak/ak_build.py:228]: if self.directory is False `ak build` failed

```
  File "/home/dav/dvp/py/ak/ak/ak_build.py", line 228, in main
    force_directory = get_repo_key_from_spec(self.directory) 
  File "/home/dav/dvp/py/ak/ak/ak_build.py", line 60, in get_repo_key_from_spec
    elif key[0:2] == './':
TypeError: 'NoneType' object is not subscriptable

```

It works for you ?

Comment (hparfr) [reply ak/ak_build.py:228]: What is the spec ?

Comment (bealdav) [reply ak/ak_build.py:228]:
```
bank-statement-paypal:
    modules:
        - account_bank_statement_import_paypal
    src: https://github.com/akretion/bank-statement-import 8.0-paypal

```

Comment (bealdav) [reply ak/ak_build.py:228]: Proposed fix https://github.com/akretion/ak/pull/76

**Pull request merged**